packages/hooks/src/__tests__/useOAuthConnectedApps.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useQuery } from '@deriv/api';
+import useOAuthConnectedApps from '../useOAuthConnectedApps';
+
+const mock_oauth_apps = [
```

Comment:
can we add types for mock_oauth_apps?

Reply:
do we really need to explicitly mention type of mock data in test file?

Reply:
I believe so, in order to avoid any TS errors in codebase. 
I've seen a lot of tests, where mocked data passed with wrong values because of absence of types.
so the project will have TS errors